components/update_client/BUILD.gn

```diff
@@ -119,6 +126,13 @@ static_library("test_support") {
     "unzip/unzip_impl_cobalt.h",
   ]
 
+  if (use_hacky_cobalt_changes) {
+    sources -= [
+      "net/network_impl_cobalt.cc",
```

Comment:
This was added in #849. We should ask the author why the file doesn't appear to be used.

Reply:
Created b/328299604 to follow up